src/app/app.module.ts

```diff
@@ -45,6 +46,7 @@ export function tokenGetter() {
     AppRoutingModule,
     CoreModule,
     SharedModule,
+    NgxEchartsModule,
```

Comment:
Do not import `NgxEchartsModule` in `app.module.ts`, instead, you can import `SharedModule` in your module to get functionalities from `NgxEchartsModule`.